src/ButtonGroup.js

```diff
@@ -35,7 +35,7 @@ const ButtonGroup = (props) => {
   ), cssModule);
 
   return (
-    <Tag {...attributes} className={classes} />
+    <Tag {...attributes} cssModule={cssModule} className={classes} />
```

Comment:
Here the default `Tag` is `div`, in which case the `cssModules` prop would not want to be passed down